tests/test_environments/test_all_environments.py

```diff
@@ -2,7 +2,7 @@
 Test all environments with random policies.
 
 This runs some basic sanity checks on the environment, namely, checking that:
-    - robot-state exists in the obs, and is a flat array
+    - proprio-state exists in the obs, and is a flat array
```

Comment:
is there a good reason for changing from "robot-state" to "proprio-state"?

Reply:
this change directly stemmed from the new `modality` attribute associated with a given `Observable` / `sensor` -- when the `obs_dict` is constructed after an `env.step()` call, individual keys are also concatenated together based on their associated sensor's `modality`.

So it seemed to me that defining the modality of the robot sensors as `proprio` vs. `robot` was more semantically meaningful and accurate, not to mention that the final modality keys `robot0_proprio-state` seems to be more informational vs what we had before (`robot0_robot-state`).

Reply:
that's fair